SPECS/xapi.spec

```diff
@@ -1,15 +1,17 @@
-%global package_speccommit 00da470e0adcf10bb7e9a118e098672766dbf4f6
-%global package_srccommit v1.249.36
+%global package_speccommit de5edecdf88cb9c615c48fedc3ad826ce85dc106
+%{!?xsrel: %global xsrel 1}
+%global package_srccommit v1.249.38
+
```

Comment:
does the extra empty line comes from upstream?

Reply:
Which extra empty line?
I don't see differences with the file from Citrix. Can you be more specific on the line you're talking please?

If it's between line 3 and the comment on line 5, yes it's coming from upstream.